src/audio/detect_test.c

```diff
@@ -544,7 +544,7 @@ static int test_keyword_get_config(struct comp_dev *dev,
 
 	/* Copy back to user space */
 	bs = cd->config.size;
-	trace_value(bs);
+	trace_keyword("value of bs: %u", bs);
```

Comment:
What is "bs" ? Best to use "block size " if bs represent block size.

Reply:
changed